interfaces/cython/cantera/reactor.pyx

```diff
@@ -1071,6 +1133,48 @@ cdef class ReactorNet:
         self.net.getState(&y[0])
         return y
 
+    def get_derivative(self, k):
+        """
+        Get the k-th time derivative of the state vector of the reactor network.
+        """
+        if not self.n_vars:
+            raise CanteraError('ReactorNet empty or not initialized.')
+        cdef np.ndarray[np.double_t, ndim = 1] dky = np.zeros(self.n_vars)
+        self.net.getDerivative(k, & dky[0])
+        return dky
+
+    property last_order:
```

Comment:
Could be removed from cython

Reply:
I don't see a use for this outside the estimation routine. Do you have one in mind? If not, I would be in favor of removing.

Reply:
Eliminated.